plat/kvm/Config.uk

```diff
@@ -9,6 +9,7 @@ menuconfig PLAT_KVM
        select LIBFDT if ARCH_ARM_64
        select ELF64_TO_32 if ARCH_X86_64
        select LIBUKRELOC
+       select LIBUKINTCTLR
```

Comment:
Below you can have each VMM select the GIC version it uses by default:
``` 
 config KVM_VMM_QEMU
        bool "QEMU"
+       imply LIBUKINTCTLR_GICV2
        imply LIBUKTTY_PL011
        imply LIBUKRTC_PL031
 
 config KVM_VMM_FIRECRACKER
        bool "Firecracker"
        select KVM_BOOT_PROTO_LXBOOT
+       imply LIBUKINTCTLR_GICV3
        imply LIBUKTTY_NS16550
        imply LIBUKRTC_PL031
        depends on ARCH_X86_64 || ARCH_ARM_64
```
And since `LIBUKINTCTLR` is selected by each GIC version option in `drivers/ukintctlr/gic/Config.uk`, you can remove `select LIBUKINTCTLR` from above.


Also `UKPLAT_ALLOW_GIC` is not required:
```
@@ -225,8 +227,4 @@ config VIRTIO_9P
               Virtio 9P driver.
 endmenu
 
-config UKPLAT_ALLOW_GIC
-       bool
-       default y
-
 endif
```